docs/aerodromes/classc/Edinburgh.md

```diff
@@ -4,7 +4,74 @@
\ No newline at end of file
+| Name            | Callsign           | Frequency  | Login Identifier |
+| --------------- | ------------------ | ---------- | ---------------- |
+| Edinburgh ADC   | Edinburgh Tower    | 118.300    | ED_TWR           |
+| Edinburgh SMC   | Edinburgh Ground   | 134.100    | ED_GND           |
+
+## Airspace
+<figure markdown>
+![EDN ADC Airspace](img/edncircuits.png){ width="700" }
+</figure>
+
+- **EDN ADC Airspace**: `SFC` to `A015` within the Edinburgh Control Zone (ED CTR).
+
+## Runway Modes
+Single runway operations only.
+
+## Circuit Procedures
+
+The Edinburgh Circuit Area (EDN CIRA) is active at all times when the Edinburgh CTR is active.
+
+Circuit operations occur within the CTR bounded by the Gawler River to the north and the powerline running from the Gawler River (1.5NM south of the Gawler Township) to the south eastern boundary of the ED CTR. Vertical Limits `SFC` to `A020`
+
+If Parafield RWY 03L is active, it can infringe on the circuit deadside. VFR traffic in the ECA is instructed to remain on the west of the RWY centreline.
+
+**Example**:
+- **EDN ADC**: "CIVIL TRANSIT IN PROGRESS, DEAD-SIDE NOT AVAILABLE"
+
+### Circuit Heights
+| Type    | Normal Altitude |
+| ------- | --------------- |
+| Non Jet | A015            |
+| Jet     | A020            |
+
+
+### Circuit Direction
+
+| Runway | Direction |
+| ------ | --------- |
+| 18     | Left      |
+| 36     | Left      |
+
+## Departure Levels
+
+### Departures
+Aircraft can expect to depart to the north, regardless of the duty runway, to conform with Adelaide traffic.
+
+Aircraft will be cleared to an initial level of A040 at clearance delivery unless a lower level is requested. Adelaide Approach will clear aircraft to their planned height following departure.
+
+Aircraft departing YPED into EDN TCU Class C airspace will be coordinated from EDN ADC at taxi.
+
+!!! example
+    <span class="hotline">**EDN ADC** -> **AD TCU**</span>: "Taxi, BLKT11 for YMES, via GULLY"    
+    <span class="hotline">**AD TCU** -> **EDN ADC**</span>: "BLKT11"
+
+When the aircraft is ready for takeoff, EDN ADC will give a "Next" call, where AD TCU will provide the cleared level.
+
+!!! example
+    <span class="hotline">**EDN ADC** -> **AD TCU**</span>: "Next, BLKT11, runway 18"    
+    <span class="hotline">**AD TCU** -> **EDN ADC**</span>: "BLKT11, A050"    
+    <span class="hotline">**EDN ADC** -> **AD TCU**</span>: "A050, BLKT11"    
+
+### Arrivals
+AD TCU must coordinate the sequence information to EDN ADC prior to transfer of jurisdiction.
+
+!!! example
+    <span class="hotline">**AD TCU** -> **EDN ADC**</span>: "New sequence of 2. MRNR21, via AD, Number 1. MRNR22, via AD, Number 2"    
+    <span class="hotline">**EDN ADC** -> **AD TCU**</span>: "MRNR21, Number 1. MRNR22, Number 2"
+
+## Helicopter Operations
```

Comment:
Could we please move this above the coord section to be in line with other tower formatting?

Reply:
It's still showing below the Coord section. Could you please move it above the coord section?